packages/indy-vdr/tests/setup.ts

```diff
@@ -1,6 +1,3 @@
-// Needed to register indy-vdr node bindings
-import '../src/index'
-
-require('@hyperledger/indy-vdr-nodejs')
+import '@hyperledger/indy-vdr-nodejs'
```

Comment:
These imports in tests' setup.ts are not needed anymore, are they?

Reply:
They are! Some tests don't use the module (e.g. `IndyVdrSovDidResolver.test.ts`), but do use the binary. It will result in the binary not being registered on the shared package if you run the test (mainly in isolation).

I first started importing the nodejs bindings in these specific files, but I thought why make it more complex than needs to be and just imported it in the setup